src/governance-vote.h

```diff
@@ -68,6 +68,8 @@ enum vote_signal_enum_t  {
     VOTE_SIGNAL_CUSTOM20   = 35
 };
 
+static const int MAX_SUPPORTED_VOTE_SIGNAL = 4;
```

Comment:
> I see no issues using smth that is defined in language specification

C++ is a very underspecified, underconstrained language.  It's possible to write code that conforms to the spec but is insanely difficult for a human to interpret.

My approach is to keep things as simple and idiomatic as possible.  I guess it's basically to write C++ code as if it were Ada.